provider/cmd/pulumi-resource-purrl/schema.json

```diff
@@ -187,11 +201,11 @@
                     "type": "string"
                 },
                 "deleteResponse": {
-                    "description": "The response from the API call.",
+                    "description": "The response from the (delete) API call.",
                     "type": "string"
                 },
                 "deleteResponseCodes": {
-                    "description": "The expected response code.",
+                    "description": "The expected response code(s) for deletion. Deprecated -- use `expectedDeleteResponseCodes` instead.",
```

Comment:
Hi @necaris,

can you use `deprecationMessage` instead?

see  https://www.pulumi.com/docs/guides/pulumi-packages/schema/#property